admin/partials/engagement-cloud-admin-display.php

```diff
@@ -66,8 +66,7 @@ public function __construct( $plugin_name, $webapp_url ) {
 	 */
 	public function display_plugin_setup_page() {
 
-		$charset = get_bloginfo('charset');
-		$store_name = html_entity_decode(get_bloginfo('name'), ENT_QUOTES, empty($charset) ? 'UTF-8' : $charset);
+		$store_name = html_entity_decode(get_bloginfo('name'), ENT_QUOTES, $this->get_charset());
```

Comment:
Great refactor! :+1: Makes me think we could push the `html_entity_decode` call into a `get_store_name` method?